modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -310,7 +318,22 @@ public void testSendNotification() throws Exception {
 			false,
 			StringBundler.concat(
 				user1.getEmailAddress(), StringPool.COMMA,
-				user2.getEmailAddress()));
+				user2.getEmailAddress()),
+			ObjectDefinitionConstants.SCOPE_COMPANY);
+
+		// Multiples emails for each main recipient with a ";" separator
```

Comment:
Here we need to put the comments in alphabetical order. I'm not sure that would be the ideal comment either.

Reply:
Thanks for reminding me of this!